website/blog/2023-07-19-silver.md

```diff
@@ -0,0 +1,817 @@
+done by [jkciesluk](https://github.com/jkciesluk), you will be able to run
+`metals.reset-workspace` command to remove all the possible caches and compile
+artifact that might be causing issues in your workspace.
+
+This should be seen as a last resort and we hope that anyway if you keep having
+issues you will report them to us so we can properly fix them.
+
+## Better support for Scala CLI
+
+With Scala CLI becoming the official runner for Scala 3, we have been working on
+improving the support for it in Metals. This release brings a number of
+improvements that aim to improve both Scala CLI experience and the overall Scala
+experience.
+
+This includes the following:
+
+- If there are no build tools present Metals will now fallback to Scala CLI,
+  which means that even if a users creates a single file they will be able to
+  run it and get all the proper support users would expect. If a build tool
+  configuration appears in the workspace, Metals will suggest switching to that
+  instead.
+- Scala CLI gets the same status as the already supported build tools, which
+  means that we will not automatically suggest importing another build tool if
+  we have Scala CLI already configured.
+- If `project.scala` is present in the workspace, Metals will suggest importing
+  Scala CLI project, which includes automatically using the `setupIde` command
+  to setup the .bsp configuration.
+
+This splendid work was done by [kasiaMarek](https://github.com/kasiaMarek)
+
+## Support new Scala versions
+
+### Support for Scala 3.3.0
+
+For more information about the release take a look at
+[https://www.scala-lang.org/blog/2023/05/30/scala-3.3.0-released.html](https://www.scala-lang.org/blog/2023/05/30/scala-3.3.0-released.html)
+
+Here we only show a short summary of the main feature from the release notes.
+
+#### Linting
+
+Scala 3.3.0 brings back linting to the Scala compiler. Right now, you can enable
+checking for unused symbols and discarded values. More linting options will come
+soon in the following Scala 3.3.x releases.
+
+#### Fewer braces
+
+Braces around method parameters can now be replaced with a colon. This can lead
+to cleaner, shorter, and often more readable code in places like configuration
+DSLs or test case definitions.
+
+#### Boundary and break
+
+Two new methods were added to the standard library: boundary and break. They are
+safer and more expressive replacements for non-local returns, which were
+deprecated recently.
+
+break allows for a type-safe early escape from anywhere inside the block
+delimited by boundary to its end, returning the passed value from the entire
+block.
+
+#### The new default implementation of lazy vals
+
+Last but not least, we have changed the default implementation of lazy vals. The
+new implementation has better performance and is safer under parallel access.
+This may result in improvements in the performance of effect systems.
```

Comment:
No strong feeling on this, but I don't actually think this is needed. It might even be confusing that we're outlining features of the Scala release here in the Metals release. I'd focus on what this release of Metals brings you and link to the relevant places elsewhere. I'm not sure this add a lot to the Metals release notes.